### DISCUSSION TITLE
Button Send Form

### DISCUSSION THREAD

**Discussion by cfuent-es** [livewire/livewire#4257] [1. Help] [answered]
Hello,

I have an address registration component. I'm using a button to search a public function, but this button search is also activating the save (even though it's not a type="submmit" button). How could I disable this form submission?

Component: https://pastebin.com/kJRL4BAt
Blade: https://pastebin.com/QBpDUSyR (search button on line 15)

Thanks!

**Comment by CorySheppard** [comment, marked as answer]
You need to specify that it's just a normal button with `type="button"`. By default it will continue to submit the form it's in.

**Reply by joshhanley** [reply]
@fuentesbr html buttons by default are submit buttons, so you don't need to specify `type="submit"`. 

So what @CorySheppard suggested above is how you would fix it, by ensuring the button is set to `type="button"`.

Hope this helps!

**Reply by cfuent-es** [reply]
Thanks!!!

**Comment by jonneroelofs** [comment]
Change it to wire:click.stop , then the event won't bubble up in the DOM and the form should not save.

https://laravel-livewire.com/docs/2.x/actions#event-modifiers
